tools/harness-thci/OpenThread_WpanCtl.py

```diff
@@ -0,0 +1,2598 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2018, The OpenThread Authors.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. Neither the name of the copyright holder nor the
+#    names of its contributors may be used to endorse or promote products
+#    derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+"""
+>> Thread Host Controller Interface
+>> Device : OpenThread_WpanCtl THCI
+>> Class : OpenThread_WpanCtl
+"""
+
+import re
+import time
+import socket
+import logging
+from Queue import Queue
+
+import serial
+from IThci import IThci
+from GRLLibs.UtilityModules.Test import Thread_Device_Role, Device_Data_Requirement, MacType
+from GRLLibs.UtilityModules.enums import PlatformDiagnosticPacket_Direction, PlatformDiagnosticPacket_Type
+from GRLLibs.UtilityModules.ModuleHelper import ModuleHelper, ThreadRunner
+from GRLLibs.ThreadPacket.PlatformPackets import PlatformDiagnosticPacket, PlatformPackets
+from GRLLibs.UtilityModules.Plugins.AES_CMAC import Thread_PBKDF2
+
+"""wpanctl carrier info and wpanctl command prefix"""
+WPAN_CARRIER_USER = 'pi'
+WPAN_CARRIER_PASSWD = 'raspberry'
+WPAN_CARRIER_PROMPT = 'pi@raspberrypi'
+WPAN_INTERFACE = 'wpan0'
+WPANCTL_CMD = 'sudo wpanctl -I ' + WPAN_INTERFACE + ' '
+
+"""regex: used to split lines"""
+LINESEPX = re.compile(r'\r\n|\n')
+
+class OpenThread_WpanCtl(IThci):
+    LOWEST_POSSIBLE_PARTITION_ID = 0x1
```

Comment:
please revert and keep the typo `LOWEST_POSSIBLE_PARTATION_ID` as it is an constant which will be used by test script of TH.